plugins/woocommerce/src/StoreApi/Utilities/CartController.php

```diff
@@ -1378,6 +1378,21 @@ protected function sanitize_variation_data( $variation_data, $variable_product_a
 			$lowercase_attribute_label = strtolower( $attribute_label );
```

Comment:
I suppose for a small perf gain, this could be moved below `if` added in this PR. Makes sense to define things closer to where they're used in my opinion.

Reply:
@opr that's fair - I think I was trying to modify the least number of lines :D But I agree with you! Changed in c17b4a2ae4